src/unix/fs.c

```diff
@@ -49,6 +49,7 @@
     defined(__FreeBSD__)    ||                                            \
     defined(__OpenBSD__)    ||                                            \
     defined(__NetBSD__)
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
@TheAlphaNerd … doesn’t that CI run imply that BSD *does* have the macros there?

I think there should be just an `#include <limits.h>` where the other includes are. On systems that are not FreeBSD, things just worked because one of the other system headers includes `limits.h` itself.

On my Linux machine, the chain is `uv.h -> uv-unix.h -> dirent.h -> limits.h`, and I guess something similar is going on for OS X.